src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SwaggerRoutes.scala

```diff
@@ -19,7 +19,7 @@ import scala.util.{Failure, Success, Try}
   * Created by dvoet on 7/18/17.
   */
 trait SwaggerRoutes extends LazyLogging {
-  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/4.1.2"
+  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/4.5.2"
```

Comment:
and dunno how refactor-y you want to get, but https://github.com/broadinstitute/rawls/blob/ea0348ac456fdaac7dd1a28225e7627d15fbaa7e/core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/SwaggerRoutes.scala#L16 might also be helpful … it prevents always having to update the version in two places

Reply:
I would prefer not to refactor that as I go along this time, hoping to get the security stuff done with to get back to the billing resolution ASAP